hikari/impl/bot.py

```diff
@@ -526,12 +574,12 @@ def run(
             The number of shards to use in the entire distributed application.
             Defaults to `builtins.None` which results in the count being
             determined dynamically on startup.
-        status : hikari.presences.Status
-            The initial status to show for the user presence on startup.
-            Defaults to `hikari.presences.Status.ONLINE`.
         """
+        if shard_ids is not None and shard_count is None:
+            raise RuntimeError("'shard_ids' must be passed with 'shard_count'")
```

Comment:
This should be a TypeError come to think of it...

Reply:
Will change :+1: